data/ka'het/ka'het outfits.txt

```diff
@@ -223,3 +223,87 @@ outfit "Ka'het Reserve Accumulator"
 	"outfit space" -3
 	"energy capacity" 600
 	description "This tiny battery was developed for use by the Ka'mar drones as an emergency energy storage, in case their generators can't quite suffice for the high power demand of most Builder systems."
+
+# Engines:
+
+outfit "Maeri Engine Nacelles"
+	category "Engines"
+	"cost" 928000
+	thumbnail "outfit/ka'het maeri engine"
+	"mass" 52
+	"outfit space" -76
+	"engine capacity" -59
+	"energy consumption" 5.01
+	"heat generation" 1.1
+	"thrust" 37.1
+	"thrusting heat" 2.8
+	"turn" 931.4
+	"turning heat" 1.3
+	"flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"steering flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"flare sound" "plasma medium"
+	description "The Ka'het engines are peculiar in that they need to be continuously charged with large amounts of energy: while this means that less power remains available for other subsystems, it is also somewhat more efficient than most types of propulsions, and generates only a tiny amount of heat."
+
+outfit "Ka'het Sustainer Nacelles"
+	category "Engines"
+	"cost" 357000
+	thumbnail "outfit/ka'het sustainer engine"
+	"mass" 27
+	"outfit space" -34
+	"engine capacity" -27
+	"heat generation" 0.6
+	"energy consumption" 2.49
+	"thrust" 15.1
+	"thrusting heat" 1.0
+	"turn" 432.1
+	"turning heat" 0.5
+	"flare sprite" "effect/ka'het flare/medium"
+		"frame rate" 30
+	"steering flare sprite" "effect/ka'het flare/medium"
+		"frame rate" 30
+	"flare sound" "plasma small"
+	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maintain its thrusters and steerings ready."
+
+outfit "Telis Engine Nacelles"
+	category "Engines"
+	"cost" 1372000
+	thumbnail "outfit/ka'het telis engine"
+	"mass" 81
+	"outfit space" -109
+	"engine capacity" -81
+	"energy consumption" 6.81
+	"heat generation" 1.4
+	"thrust" 55.4
+	"thrusting heat" 3.5
+	"turn" 1379.2
+	"turning heat" 1.9
+	"flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"steering flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"flare sound" "plasma medium"
+	description "These nacelles used to comprise the entire engine block of a Telis'het. While their size is not impressive compared to other engines, if used in conjunction with a sustainer or two, they can be a powerful form of propulsion, combining both turning and thrusting engines in one."
+
+outfit "Ka'het Compact Engine"
```

Comment:
Should this be Ka'mar instead of Ka'het?

Reply:
The MHD and support battery are still called "Ka'het" as the rest of their stuff, so to avoid confusion I'd rather call them all Ka'het